octoprint_mrbeam/static/js/app/view-models/convert.js

```diff
@@ -37,7 +37,12 @@ $(function () {
                 lineDistance: 1.0,
             },
             min: {
-                speed: 30,
+                speed:
+                    MRBEAM_LASER_HEAD_MODEL === "x"
+                        ? 100
+                        : MRBEAM_LASER_HEAD_MODEL === "S"
+                        ? 50
+                        : 30,
```

Comment:
Do You think we can have this in the laserhead profile? Or at least define it as constants at the beginning of the document

Reply:
Done